website/static/js/pointers.js

```diff
@@ -63,9 +64,10 @@ var AddPointerViewModel = oop.extend(Paginator, {
                 self.currentPage(result.page);
                 self.numberOfPages(result.pages);
                 self.addNewPaginators();
-            }).fail(
-                osfHelpers.handleJSONError
-            );
+            }).fail(function() {
+                self.elm.modal('hide');
+                osfHelpers.handleJSONError();
```

Comment:
`handleJSONError` takes the `xhr` as its first argument. You should pass the `xhr` from the callback to the `handleJSONError` function.

```js
.fail(function(xhr) {
    osfHelpers.handleJSONError(xhr);
}
```